website/docs/reference/resource-configs/teradata-configs.md

```diff
@@ -289,7 +312,112 @@ Another e.g. for adding multiple grants:
   ```
 > :information_source: `copy_grants` is not supported in Teradata.
 
-More on Grants can be found at https://docs.getdbt.com/reference/resource-configs/grants
+Refer to [grants](/reference/resource-configs/grants) for more information on Grants.
+
+## Query Band
+Query Band in dbt-teradata can be set on three levels:
```

Comment:
adjusted this section as the formatting was off

<img width="812" alt="Screenshot 2024-07-02 at 12 22 43" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/7bd654df-0b18-489f-b295-e89c1d0035e8">